.github/workflows/label-github-issues.yml

```diff
@@ -0,0 +1,40 @@
+# Adds metadata and labels to issues based on context
+# https://github.com/airbytehq/workflow-actions/blob/main/src/bin_issue.ts <-- this gets run
+
+name: "[safe to merge when failing] Add metadata and labels to issues based on context"
+on:
+  issues:
+    types: [opened, labeled, unlabeled]
+
+jobs:
+  find_valid_pat:
+    name: "Find a PAT with room for actions"
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    outputs:
+      pat: ${{ steps.variables.outputs.pat }}
+    steps:
+      - name: UUID ${{ github.event.inputs.uuid }}
+        run: true
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Check PAT rate limits
+        id: variables
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.OCTAVIA_PAT }} \
+            ${{ secrets.OCTAVIA_4_ROOT_ACCESS }}
+  shared-issues:
+    name: "Add Labels to Issues.  Safe to Merge on fail"
+    runs-on: ubuntu-latest
+    needs:
+      - find_valid_pat
+    steps:
+      - uses: nick-fields/private-action-loader@v3
+        with:
+          pal-repo-token: "${{ secrets.OCTAVIA_PAT }}"
```

Comment:
Can we use the find_valid_pat output token for this as well?

Reply:
ty pete.  missed it